packages/core/types/zapier.custom.d.ts

```diff
@@ -37,13 +37,71 @@ export interface Bundle<InputData = { [x: string]: any }> {
   inputDataRaw: { [x: string]: string };
   meta: {
     isBulkRead: boolean;
+
+    /**
+     * If true, this poll is being used to populate a dynamic dropdown.
+     * You only need to return the fields you specified (such as id and
+     * name), though returning everything is fine too.
+     */
     isFillingDynamicDropdown: boolean;
+
+    /**
+     * If true, this run was initiated manually via the Zap Editor.
+     */
     isLoadingSample: boolean;
+
+    /**
+     * If true, the results of this poll will be used to initialize the
+     * deduplication list rather than trigger a zap. You should grab as
+     * many items as possible.
+     */
     isPopulatingDedupe: boolean;
+
+    /**
+     * (legacy property) If true, the poll was triggered by a user
+     * testing their account (via clicking “test” or during setup). We
+     * use this data to populate the auth label, but it’s mostly used to
+     * verify we made a successful authenticated request
+     *
+     * @deprecated
+     */
     isTestingAuth: boolean;
+
+    /**
+     * The number of items you should fetch. -1 indicates there’s no
+     * limit. Build this into your calls insofar as you are able.
+     */
     limit: number;
+
+    /**
+     * Used in paging to uniquely identify which page of results should
+     * be returned.
+     */
     page: number;
-    zap?: { id: string };
+
+    /**
+     * When a create is called as part of a search-or-create step,
+     * this will be the key of the search.
+     */
+    withSearch: string | undefined;
```

Comment:
```suggestion
    withSearch?: string;
```

The backend code shows that it is not always present.